shared/js/ui/models/whitelist.es6.js

```diff
@@ -21,6 +22,22 @@ Whitelist.prototype = window.$.extend({},
         value: false
       }
       })
+    },
+
+    addDomain: function (url) {
+      const parsedUrl = tldjs.tldExists(url) ? tldjs.parse(url) : ''
```

Comment:
Not sure if `tldExists` is the best method to use. From the [readme](https://www.npmjs.com/package/tldjs#tldexists):

```js
tldExists('com');             // returns `true`
tldExists('uk');              // returns `true`
```

Might as well just parse the URL, we're checking it on L29 anyway.

Reply:
@andrey-p I should have left a comment here actually, but I'm using that method because otherwise random alphanumeric strings get accepted too - eg, without this check, strings like "asdasd" will be considered valid and added to the whitelist, while what we want is in the format "asdasd.com".

Also from the readme:
```js
tldExists('google.com');      // returns `true`
tldExists('google.local');    // returns `false` (not an explicit registered TLD)
```

Reply:
Ohh got it, I see. I didn't expect `tldjs` to work this way.

Yeah, a code comment would be excellent here.